docs/guide/sending-email.md

```diff
@@ -0,0 +1,226 @@
+
+::: note
+Note that in the Nunjucks template, the area is passed in through the `data` context. It isn't being passed in through the `data.piece` or `data.page` context as it would for a normal Nunjucks template view.
+:::
+
+## Triggering email from a route
+
+In addition to using `handlers()` to trigger email delivery, you can use `apiRoutes()`. This can be used to send an email when a user gets redirected to a selected page or sends a POST request, like a form submission. All of the same arguments passed when invoking the `email()` method from a module need to be supplied when using the method in a route. In this case, we need to create options and our data object from our `req.body`.
+
+### Example usage
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  extend: '@apostrophecms/widget-type',
+  options: {
+    lable: 'contact'
+  },
+  ...,
+  apiRoutes(self) {
+    return {
+      post: {
+        submit: async function (req) {
+          if (!req.body.email || !req.body.name) {
+            throw self.apos.error('invalid');
+          }
+          const email = self.apos.launder(req.body.email);
+          const options = {
+            from: 'admin@mysite.com',
+            to: email,
+            subject: 'Thanks for your submission!'
+          };
+          const data = {
+            name: self.apos.launder(req.body.name)
+          };
+          const template = 'email.html';
+          try {
+            await self.email(req, template, data, options);
+            return null;
+          } catch (err) {
+            self.apos.util.error('email notification error: ', err);
+          }
+        }
+      }
+    };
+  }
+};
+
+```
+
+<template v-slot:caption>
+/modules/article/index.js
+</template>
+
+</AposCodeBlock>
+
+In this case, the `email.html` template is the same as the one used for mail delivery from our `handlers()`.
+
+So, what is going on with this code? First, we are passing our `apiRoutes()` a `post` object. This contains the functions that should be used with a `POST` HTTP request. Each expected routes should get a separate function. In this case, we are passing the function through the `submit` property. This will monitor for a POST request to `https://www.mysite.com/api/v1/submit`. We could give the property a name prefixed with a slash to monitor that exact route - `https://www.mysite.com/submit`. This is useful for when the user is being redirected to a new page. See the [reference documentation](../reference/module-api/module-overview.html#naming-routes) for more details.
+
+In the next block of code, we test to ensure that the information needed to construct the `options` and `data` arguments exists in the submission. If it does exist, the values for both those parameters are created with sanitization using `self.apos.launder()`. Finally, this information and the email template are passed to the `email()` method in a try...catch block.
```

Comment:
`launder.string`

I think we might have a `launder.email`, take a look first though

Reply:
No `launder.email`, so using `launder.string`.